data/tcp-listener.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @title TCP Listener: Ping
+ * @difficulty intermediate
+ * @tags cli
+ * @run --allow-net <url>
+ * @resource {https://deno.land/api?s=Deno.listen} Deno.listen: API
+ *
+ * An example of a TCP listener on localhost that will log the message if written to and close the connection if connected to.
+ */
+const decoder = new TextDecoder();
+
+const listener = Deno.listen({
+  hostname: "127.0.0.1",
+  port: 8080,
+  transport: "tcp",
+});
+for await (const conn of listener) {
+  const buf = new Uint8Array(1024);
```

Comment:
1024 feels arbitrary here. I would include a small section explaining why 1024 and when you should increase / decrease it.

Reply:
Based on the title of TCP Listener: Ping 1024 was chosen as a Base2 (2^10) length that would compensate for small variation in experimentation on the connecting client sending a "Ping" within reason. The size allows for some user error when connecting and throwing an overtly large "Ping" to learn by example, which seems to be the size of the repository.

IMO, getting into the details of buffer sizes, TCP window sizing, and working in high performance TCP networking falls outside the scope of denobyexample.

Reply:
I feel like a one sentence explanation of what a buffer is in this context would be "good enough", but not elaborating at all feels wrong to me.